docs/conf.py

```diff
@@ -112,7 +112,7 @@ def setup(sphinx):
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
Oh nice, so as far as I understand this ensures that locally generated style is the same as on RTD?